supportedBackends/sfs/src/test/scala/cromwell/backend/sfs/SharedFileSystemJobExecutionActorSpec.scala

```diff
@@ -1,430 +0,0 @@
-        realJsonInputFile.toFile should exist
-
-        localizedCallInputFile.isSymbolicLink shouldBe isSymlink
-        val realCallInputFile =
-          if (isSymlink) localizedCallInputFile.symbolicLink.get else localizedCallInputFile
-        realCallInputFile.toFile should exist
-      }
-    }
-  }
-
-  it should "execute calls with input files and localize them appropriately" in {
-    localizationSpec(docker = false)
-  }
-
-  it should "execute calls with input files and localize them appropriately (in Docker)" taggedAs DockerTest in {
-    localizationSpec(docker = true)
-  }
-
-  it should "abort a job and kill a process" in {
-    val workflowDescriptor = buildWdlWorkflowDescriptor(Sleep20)
-    val jobDescriptor: BackendJobDescriptor = jobDescriptorFromSingleCallWorkflow(workflowDescriptor,
-                                                                                  Map.empty,
-                                                                                  WorkflowOptions.empty,
-                                                                                  runtimeAttributeDefinitions
-    )
-    val backendRef = createBackendRef(jobDescriptor, TestConfig.backendRuntimeConfigDescriptor)
-    val backend = backendRef.underlyingActor
-
-    val execute = backend.execute
-    backendRef ! AbortJobCommand
-
-    whenReady(execute) { executionResponse =>
-      executionResponse shouldBe a[JobAbortedResponse]
-    }
-  }
-
-  def recoverSpec(completed: Boolean, writeReturnCode: Boolean = true): Assertion = {
-    val workflowDescriptor = buildWdlWorkflowDescriptor(HelloWorld)
-    val jobDescriptor: BackendJobDescriptor = jobDescriptorFromSingleCallWorkflow(workflowDescriptor,
-                                                                                  Map.empty,
-                                                                                  WorkflowOptions.empty,
-                                                                                  runtimeAttributeDefinitions
-    )
-    val backendRef = createBackendRef(jobDescriptor, TestConfig.backendRuntimeConfigDescriptor)
-    val backend = backendRef.underlyingActor
-
-    val jobPaths = JobPathsWithDocker(jobDescriptor.key, workflowDescriptor, ConfigFactory.empty)
-    jobPaths.callExecutionRoot.createPermissionedDirectories()
-    jobPaths.stdout.write("Hello stubby ! ")
-    jobPaths.stderr.touch()
-
-    val pid =
-      if (completed) {
-        if (writeReturnCode)
-          jobPaths.returnCode.write("0")
-        "0"
-      } else {
-        import sys.process._
-        val proc = Seq("bash", "-c", s"sleep 2; echo 0 > ${jobPaths.returnCode}").run()
-        val pField = proc.getClass.getDeclaredField("p")
-        pField.setAccessible(true)
-        val p = pField.get(proc)
-        val pidField = p.getClass.getDeclaredField("pid")
-        pidField.setAccessible(true)
-        pidField.get(p).toString
```

Comment:
This test relies on reflection in a way that does not work in 17.

We're already testing SFS a.k.a. Slurm in Centaur, which is preferred anyway.